spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -185,8 +195,10 @@
   context 'category and type search results' do
     before(:each) do
       create(:version, :production)
-      create(:institution, :in_nyc)
-      create(:institution, :ca_employer)
+      create(:institution, :in_nyc, approved: true)
+      create(:institution, :ca_employer, approved: true)
+      # adding a non approved institutions row
+      create(:institution, :contains_harv, approved: false)
```

Comment:
You can remove lines 200 and 201 because adding this record does not change testing for `category and type search results` either way.